README.md

```diff
@@ -65,7 +65,19 @@ for `tslint@^5.0.0`
 }
 ```
 
-**NOTE**: To use this plugin, it'd better to also use [tslint-config-prettier](https://github.com/prettier/tslint-config-prettier) to disable all prettier-related rules, so as to avoid conflicts between existed rules.
+## When to use `tslint-plugin-prettier`
+
+#### use this package when...
+
+- you want to run prettier as a plugin of tslint and you only want prettier to report issues in `.ts` or `.tsx` files
+- Ex npm script: `"lint": "tslint -p tsconfig.json"`
+
+#### don't use this package when...
```

Comment:
Though most people won't use it this way, these use-cases are still valid. _"don't use"_ looks a bit tough, it'd be better to use something more euphemistic.